pkg/daemon/daemon.go

```diff
@@ -280,9 +280,10 @@ func (dn *Daemon) runLoginMonitor(stopCh <-chan struct{}, exitCh chan<- error) {
 		case <-stopCh:
 			return
 		case msg := <-sessionNewCh:
-			glog.Infof("Detected a new logged in session: %v", msg)
-			if err := dn.applySSHTaint(); err != nil {
-				exitCh <- fmt.Errorf("Error during taint: cannot apply taint due to: %v", err)
+			glog.Infof("Detected a new login session: %v", msg)
+			glog.Infof("Login access is discouraged! Applying annotation: %v", MachineConfigDaemonSSHAccessAnnotationKey)
+			if err := dn.nodeWriter.SetSSHAccessed(dn.kubeClient.CoreV1().Nodes(), dn.name); err != nil {
+				exitCh <- fmt.Errorf("Error: cannot apply annotation for SSH access due to: %v", err)
```

Comment:
Note this will degrade the node if we can't set the annotation. Do we really want that? (Not rhetorical, actually asking. :)). I'm leaning more towards not making this a hard error and just logging it for now.

Reply:
Good point, we can switch to a warning, but as far as I can tell all other failed nodewriter invocations also degrade the node. If we cannot write an annotation there is likely an issue with the node, and future updates, etc. would fail.

I don't think there is a specific case that an inability to write an annotation would come only from this?